modules/core/src/main/java/org/apache/ignite/mxbean/CacheMetricsMXBean.java

```diff
@@ -351,4 +351,60 @@
     @MXBeanDescription("Number of keys processed during index rebuilding. To get remaining number of keys for " +
         "rebuilding, subtract current value from cache size.")
     @Override public long getIndexRebuildKeysProcessed();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Number of already rebalanced keys.")
+    @Override public long getRebalancedKeys();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Number estimated to rebalance keys.")
+    @Override public long getEstimatedRebalancingKeys();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("The number of partitions need to be cleared before actual rebalance start.")
+    @Override public long getRebalanceClearingPartitionsLeft();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("The mean time to execute cache invokes (time in µs).")
+    @Override public float getEntryProcessorAverageInvocationTime();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("The percentage of invocations on keys, which exist in cache.")
+    @Override public float getEntryProcessorHitPercentage();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("The total number of invocations on keys, which exist in cache.")
```

Comment:
The total number of invocations on keys[comma not needed] which exist in _the_ cache.